dpypelines/pipeline/dataset_ingress_v1.py

```diff
@@ -264,24 +305,143 @@ def dataset_ingress_v1(files_dir: str, pipeline_config: dict):
                 "metadata_path": metadata_path,
             },
         )
-
     except Exception as err:
         logger.error(
-            "Error occured while running transform function",
+            "Error occurred while running transform function",
             err,
             data={
                 "transform_function": transform_function,
                 "pipeline_config": pipeline_config,
                 "args": args,
-                "kwrags": kwargs,
+                "kwargs": kwargs,
             },
         )
         de_notifier.failure()
         raise err
 
     de_notifier.success()
+
     # TODO - validate the metadata once we have a schema for it.
 
     # TODO - validate the csv once we know what we're validating
 
+    # ---------------------------------
```

Comment:
Can you delete this down to (and including) the "I worked" print statement. I got rid of this a little while ago, too much slack noise.